packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -224,10 +242,17 @@ class Plugin implements CLNPlugin {
             await _handlingSubscription(
                 event: jsonRequest.method, request: param);
           } else {
-            var result = await _call(jsonRequest.method, param);
-            var response =
-                Response(id: jsonRequest.id, result: result).toJson();
-            stdout.write(json.encode(response));
+            if (!rpcMethods.containsKey(jsonRequest.method)) {
+              var result = await _callHook(jsonRequest.method, param);
+              var response =
+                  Response(id: jsonRequest.id, result: result).toJson();
+              stdout.write(json.encode(response));
+            } else {
+              var result = await _callRPCMethod(jsonRequest.method, param);
+              var response =
+                  Response(id: jsonRequest.id, result: result).toJson();
+              stdout.write(json.encode(response));
+            }
```

Comment:
I think this change can be restored to the old one?

Reply:
I am sorry I didnt understand.